pkg/component/autoscaling/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -70,30 +72,33 @@ func New(
 	image string,
 	replicas int32,
 	config *gardencorev1beta1.ClusterAutoscaler,
+	expanderConfigMap *corev1.ConfigMap,
 	maxNodesTotal int64,
 	runtimeVersion *semver.Version,
 ) Interface {
 	return &clusterAutoscaler{
-		client:         client,
-		namespace:      namespace,
-		secretsManager: secretsManager,
-		image:          image,
-		replicas:       replicas,
-		config:         config,
-		maxNodesTotal:  maxNodesTotal,
-		runtimeVersion: runtimeVersion,
+		client:            client,
+		namespace:         namespace,
+		secretsManager:    secretsManager,
+		image:             image,
+		replicas:          replicas,
+		config:            config,
+		expanderConfigMap: expanderConfigMap,
+		maxNodesTotal:     maxNodesTotal,
+		runtimeVersion:    runtimeVersion,
 	}
 }
 
 type clusterAutoscaler struct {
-	client         client.Client
-	namespace      string
-	secretsManager secretsmanager.Interface
-	image          string
-	replicas       int32
-	config         *gardencorev1beta1.ClusterAutoscaler
-	maxNodesTotal  int64
-	runtimeVersion *semver.Version
+	client            client.Client
+	namespace         string
+	secretsManager    secretsmanager.Interface
+	image             string
+	replicas          int32
+	config            *gardencorev1beta1.ClusterAutoscaler
+	expanderConfigMap *corev1.ConfigMap
```

Comment:
I think it's enough to just pass the real configuration:

```suggestion
	expanderConfiguration *string
```